projects/lcms/Dockerfile

```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 Google Inc.
+# Copyright 2023 Google Inc.
```

Comment:
Please do not change the year of the Copyright in existing flies. There are several files with this.

Reply:
Sorry for this. It was also changed in cms_transform_extended_fuzzer.

Reply:
Thanks for fixing!